### DISCUSSION TITLE
Welcome to Landscapist Discussions!

### DISCUSSION THREAD

**Discussion by skydoves** [skydoves/landscapist#10] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by HuixingWong** [comment]
### Your code looks like poems

**Reply by sam829** [reply]
I want to take follow up on blurring image question. Like can we have something that generates a blur hash image while the image of high quality is getting downloaded?
